README.md

```diff
@@ -54,6 +54,11 @@ $ ./script/build
 $ cp hub YOUR_BIN_PATH
 ~~~
 
+# Or, if your go development bin directory is already in your $PATH:
```

Comment:
The directory where go puts it's binaries. For example, on my machines:
```
% echo $GOPATH
/Users/sonia/go
```
So compiled binaries (eg golint, go-bindata) get put in ```/Users/sonia/go/bin```. Which is also in my $PATH:
```
% echo $PATH
/usr/local/bin:/Users/sonia/bin:/Users/sonia/bin/convert:/Users/sonia/go/bin:
                                                                                               ^^^^^^^^^^^^^^^
/usr/local/go/bin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X11/bin:/usr/texbin:/usr/local/ec2/ec2-api-tools-1.7.3.0/bin
```